fone2val/README.md

```diff
@@ -12,7 +12,7 @@ The [`carTelemetry_feeder.ini`](./config/carTelemetry_feeder.ini)  contains `kuk
 
 Before starting the [F1 feeder](./carTelemetry_feeder.py), you need to start the `kuksa.val databroker` docker container by running the following command in the main project folder:
 ```
-docker run -it -v ./VSS:/VSS --rm --net=host -p 127.0.0.1:8090:8090 -e LOG_LEVEL=ALL ghcr.io/eclipse/kuksa.val/databroker:master --insecure --vss /VSS/vss.json
+docker run -it -v ./VSS:/VSS --rm --net=host -p 127.0.0.1:8090:8090 -e LOG_LEVEL=ALL ghcr.io/eclipse/kuksa.val/databroker:0.4.3 --insecure --vss /VSS/vss.json
```

Comment:
why use 0.4.3 not working with 0.5.0? which interface are you using? `sdv.databroker.v1` ? then you can re enable it with `--enable-databroker-v1`